tasks/prime.py

```diff
@@ -568,6 +587,8 @@ def update_mto_shipment_status(self, overrides=None):
             headers=headers,
             **self.user.cert_kwargs,
         )
+        if resp.status_code == 409:
+            print("QQQ ============================= \n" + "payload status: " + payload["status"] + "\n ======= shipment status:===" + mto_shipment["status"] + "mto shipment id: " + str(mto_shipment["id"]))
```

Comment:
One thing I'm seeing with the statement that I'm trying to debug is:
mto_shipment["status"] is coming back as SUBMITTED
but when I look at mto_shipment["id"] in the database, the status is APPROVED and the logic that starts on line 570 seems to be using the db status which causes an (expected) failure if the payload["status"] isn't DIVERSION_REQUESTED (the only valid status a mto_shipment can be updated to if it's current status is APPROVED)
<img width="1391" alt="Screen Shot 2021-12-15 at 5 24 03 PM" src="https://user-images.githubusercontent.com/67110378/146274746-d057a84b-9283-4a80-ae3b-34f72f82c959.png">
<img width="1388" alt="Screen Shot 2021-12-15 at 5 24 41 PM" src="https://user-images.githubusercontent.com/67110378/146274749-f0241832-7e02-43ba-9457-628075340ed8.png">

Reply:
If I am understanding this correctly, it looks like this is an issue with the local storage making it so that `mto_shipment["status"]` is giving you stale data. If we make a change similar to the change in Namibia's [PR](https://github.com/transcom/milmove_load_testing/pull/94), we should be able to get the actual status and then add the appropriate logic to either coerce the data to pass or skip the test.